framework/messages/hr/yii.php

```diff
@@ -37,7 +37,7 @@
     'Page not found.' => 'Stranica nije pronađena.',
     'Please fix the following errors:' => 'Molimo vas ispravite pogreške:',
     'Please upload a file.' => 'Molimo vas da uploadate datoteku.',
-    'Showing <b>{begin, number}-{end, number}</b> of <b>{totalCount, number}</b> {totalCount, plural, one{item} other{items}}.' => 'Prikazuj  <b>{begin, number}-{end, number}</b> od <b>{totalCount, number}</b> {totalCount, plural, =1{stavka} one{# stavka} few{# stavke} many{# stavki} other{# stavki}}.',
+    'Showing <b>{begin, number}-{end, number}</b> of <b>{totalCount, number}</b> {totalCount, plural, one{item} other{items}}.' => 'Prikazuj  <b>{begin, number}-{end, number}</b> od <b>{totalCount, number}</b> {totalCount, plural, one{ stavka} few{ stavke} many{ stavki} other{ stavki}}.',
```

Comment:
Is extra space necessary? I think it will produce `  ` (double space) like that...